doc/sphinx/proof-engine/tactics.rst

```diff
@@ -105,6 +108,277 @@ bindings_list`` where ``bindings_list`` may be of two different forms:
+For a goal :g:`A \/ B`, use :tacn:`left` to replace the current goal with :g:`A`, or
+:tacn:`right` to replace the current goal with :g:`B`.
+
+**describe [] and , forms, mention they are used but not recommended**
+
+* :n:`( {+, @simple_intropattern}` ) - matches
+  a product over an inductive type :g:`I` with a single
+  constructor that itself has at least `n` arguments.  It performs a case
+  analysis over the hypothesis, as :n:`@destruct` would, and applies the
+  patterns to the arguments of the constructor of :g:`I`.  Note that
+  that :n:`( {+, @simple_intropattern} )` is equivalent to :n:`[{+ @simple_intropattern}]`.
+
+**need example**
+
+* :n:`( {+& @simple_intropattern} )` - a shortcut for matching with
+  :n:`(@simple_intropattern,( ... ,( ..., @simple_intropattern ) ... ))`.  It expects the hypothesis to be a sequence of
+  right-associative binary inductive constructors such as :g:`conj` or
+  :g:`ex_intro`.
+  :ref:`Example <intropattern_ampersand_ex>`
+
+The other splitting patterns described below are not frequently used.
+
+These patterns can be used when the hypothesis is an equality:
+
+* :n:`->` - replaces the right-hand side of the hypothesis with the left-hand
+  side of the hypothesis in the conclusion of the goal; the hypothesis is
+  cleared; if the left-hand side of the hypothesis is a variable, it is
+  substituted everywhere in the context and the variable is removed.
+  :ref:`Example <intropattern_rarrow_ex>`
+* :n:`<-` - similar to :n:`->`, but replaces the left-hand side of the hypothesis
+  with the right-hand side of the hypothesis.
+* :n:`[= {*, @intropattern} ]` - If the product is over an equality type,
+  applies either :tacn:`injection` or :tacn:`discriminate`
+  instead of :tacn:`destruct`; if :tacn:`injection` is applicable, the intropatterns
+  are used on the hypotheses generated by :tacn:`injection`; if the
+  number of patterns is smaller than the number of hypotheses generated, the
+  pattern :n:`?` is used to complete the list.
+  :ref:`Example <intropattern_injection_ex>`
+
+* :n:`*` - introduces one or more quantified variables from the result
+  until there are no more quantified variables.
+  :ref:`Example <intropattern_star_ex>`
+
+* :n:`**` - introduces one or more quantified variables or hypotheses from the result until there are
+  no more quantified variables or implications (:g:`->`).  :g:`intros **` is equivalent
+  to :g:`intros`.
+  :ref:`Example <intropattern_2stars_ex>`
+
+* :n:`@simple_intropattern_closed {* % @term}` - first applies each of the terms
+  with the :tacn:`apply` tactic on the hypothesis to be introduced, then it applies :n:`@simple_intropattern_closed`.
+  :ref:`Example <intropattern_injection_ex>`
+
+  the on-the-fly application of lemmas: :n:`p{+ % @term}` where :n:`p`
+  itself is not a pattern for on-the-fly application of lemmas (note:
+  syntax is in experimental stage)  **huh?**
```

Comment:
Will do.  FWIW: Better to say "Anyway" here.  I don't recall ever using "Anyways", even in casual speech.

```
“Anyways” with the S is considered slang, and is a part of nonstandard, colloquial, or
informal English. Furthermore, since “anyway” is an adverb and it is impossible for
adverbs to be plural. In newspapers or in any example of formal writing, you will
almost never see the word “anyways” used.
```

Reply:
Interesting, I had read somewhere else that the two were equivalent...